test/components/Form.spec.ts

```diff
@@ -272,37 +272,65 @@ describe('Form', () => {
       ])
     })
 
-    test('touchedFields works', async () => {
+    test('touchedFields works on focus', async () => {
```

Comment:
Can we keep the original test cases to test the touched / dirty / blurred fields logic and add new ones to test the reactivity?

Reply:
@romhml Kind of confused by this - the modified tests still test the touched / dirty / blurred fields logic.

The `await flushPromises()` they have leads me to believe they were trying to test reactivity.